### PR TITLE
NAT rules for ExternalCIDR (3/4)

### PR DESCRIPTION
**Pull request by davidefalcone1**
# Description

This PR adds the RemoveMapping function. It is used to remove a NAT mapping both from the in-memory structure of the inflater and the NatMapping resource.

### CODE REVIEW

File: pkg/liqonet/ipam_test.go [hunk 1]
```diff
@@ -1288,6 +1289,14 @@ var _ = Describe("Ipam", func() {
 
 				// Check if IP is freed
 				Expect(ipamConfig.Spec.EndpointMappings).To(HaveLen(0))
+
+				// Check NatMapping resources
```
Comment (palexster) [inline pkg/liqonet/ipam_test.go:1293]: Can you comment more about what are you asserting?

File: pkg/liqonet/natmappinginflater/natMappingInflater.go [hunk 2]
```diff
@@ -263,6 +265,53 @@ func (inflater *NatMappingInflater) addOrUpdateMappingInResource(oldIP, newIP, c
 	return nil
 }
 
+// RemoveMapping removes a mapping from both resource and in-memory structure.
+func (inflater *NatMappingInflater) RemoveMapping(oldIP, clusterID string) error {
+	var exists bool
+	var mappings netv1alpha1.Mappings
+	// Check if NAT mappings have been initilized for remote cluster.
```
Comment (palexster) [inline pkg/liqonet/natmappinginflater/natMappingInflater.go:272]:
```suggestion
	// Check if NAT mappings have been initialized for remote cluster.
```

**Review by palexster — APPROVED** [review]

**Review by alacuku — APPROVED** [review]

**Pull request merged**